app/static/js/tickets/prepay-tickets.js

```diff
@@ -0,0 +1,19 @@
+$('.autofill-checkbox').click(function(){
+        var firstName = $('#firstname').val();
+        var lastname =  $('#lastname').val();
+        var email =     $('#email').val();
+        var requiredId = $(this).data('ticket-type') + '-' + this.id ;
+        if($(this).is(':checked')) {
+            $(document.getElementById(requiredId + '-holders[firstname]')).val(firstName);
```

Comment:
We use jQuery just we don't have to do `document.getElementById` 

You can just get the element by doing `$('#' + requiredId + '-holders[firstname]')`. As simple as that.